input/docs/handbook/observable-as-property-helper/index.md

```diff
@@ -1,8 +1,15 @@
-The ObservableAsPropertyHelper, which is often abbreviated OAPH, is a class that will simplify the interop between a IObservable and a property on your View Model. It will allow you to have a property which reflects the latest value that has been sent through the IObservable<T> stream. `ObservableAsPropertyHelper<T>` properties are a way to take *Observables* and convert them into *ViewModel Properties*. 
+_**tl:dr**_  `ObservableAsPropertyHelper<T>` properties are a way to take *Observables* and convert them into *ViewModel Properties*. 
+
+The ObservableAsPropertyHelper (often abbreviated to OAPH) is a class that simplifies the interop between an IObservable and a property on your View Model. It allows you to have a property which reflects the latest value that has been sent through the IObservable<T> stream. 
```

Comment:
I believe having the unrelated subtext in comma is more correct. It's a coordinating conjuction.

Reply:
Eg you put it the abbreviated oaph in brackets

Reply:
Done